WordPress/Classes/Services/ShareExtensionService.swift

```diff
@@ -1,19 +1,35 @@
 import Foundation
+import BuildSettingsKit
 import SFHFKeychainUtils
 
 @objc
 open class ShareExtensionService: NSObject {
+    private let appGroupName: String
+    private let appKeychainAccessGroup: String
+
+    @objc public convenience override init() {
```

Comment:
I wanted to use default parameter values but couldn't because this class is used from Objective-C.